TTHAnalysis/python/plotter/functions.cc

```diff
@@ -191,8 +191,347 @@ float DPhi_CMLep_Zboost(float l_pt, float l_eta, float l_phi, float l_M, float l
   return deltaPhi(l1.Phi(),Z.Phi());
 }
 
+// May 2016: This is func which takes the 4vectors of MET, L1,L2 and reconstructs 2 taus and returns the effective Z(-->tautau) mass.                                                                                       
```

Comment:
AFAIK all this stuff is deprecated and should already have been moved to separate files like functionsEWK.cc functions2LSS.cc. Also the TTH stuff. Please check with @peruzzim @folguera  
Especially triggerSF and leptonSF are in susy-ewkino/functionsSF.cc and PU weights are in susy-ewkino/functionsPUW.cc